app/assets/stylesheets/playtime.scss

```diff
@@ -71,6 +72,16 @@ h3 {
   float: right;
 }
 
+.alert {
```

Comment:
I *did* intend to make the text larger and centered. I also noticed there were some issues with the padding - presumably from conflicts with the default? I did my best to correct them, but maybe did more harm than good?